.github/workflows/deploy.yml

```
@@ -100,34 +114,18 @@ jobs:
         with:
           node-version: ${{ env.NODE }}
 
-      - name: Serve site from subpath
-        run: |
-          mkdir deploy/
-          mv dist deploy/dashboard
-          cp deploy/dashboard/index.html deploy/index.html
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v2
+        with:
+          role-to-assume: ${{ secrets.DEPLOYMENT_ROLE_ARN }}
+          role-session-name: "ghgc-dashboard-${{ needs.define-environment.outputs.env_name }}-deployment"
+          aws-region: "us-west-2"
 
       - name: Deploy to S3 Production
-        uses: jakejarvis/s3-sync-action@master
+        run: |
+          aws s3 sync ./dist s3://${{ secrets.S3_BUCKET }}
```

```suggestion
          aws s3 sync ./dist s3://${{ secrets.S3_BUCKET }}  --delete
```
Should include the delete to clear out old files